06_gpu_and_ml/comfyui/comfy_api.py

```diff
@@ -117,7 +121,15 @@ def query_comfy_via_api(workflow_data: dict, prompt: str, server_address: str):
     return image_list
 
 
-@stub.function(image=image, gpu="any")
+@stub.function(
+    image=image,
+    gpu="any",
+    mounts=[
+        modal.Mount.from_local_file(
+            comfyui_workflow_data_path, "/root/workflow_api.json"
```

Comment:
I would suggest passing the file's contents as a string argument rather than a mount here!